systemverilog-plugin/UhdmAst.cc

```diff
@@ -3825,6 +3824,12 @@ void UhdmAst::process_tf_call(AST::AstNodeType type)
             current_node->children.push_back(node);
         }
     });
+    // prefer full name of function, with scope
+    // important, when a single function is imported from a package
+    // but it calls another, not imported, function
```

Comment:
```suggestion
    // Prefer fully qualified name of a function (prefixed with a scope).
    // This is important when a single function which has been imported from a package
    // calls another function that is not imported in the calling scope.
```
Did I understand the message correctly?

Reply:
Yes, I've updated comment, it looks better.